yang/vyatta-protocols-frr-isis-v1.yang

```diff
@@ -0,0 +1,822 @@
+          }
+        }
+      }
+    }
+    leaf net {
+      type net-address;
+      configd:help "XX.XXXX. ... .XXX.XX  Network entity title (NET)";
+      description "List of OSI NET addresses for this protocol instance.";
+    }
+    leaf log-adjacency-changes {
+      type empty;
+      configd:help "Log changes to the IS-IS adjacencies in this area.";
+      description "Log changes to the IS-IS adjacencies in this area.";
+    }
+    container lsp-gen-interval {
+      configd:help "Minimum LSP regeneration interval.";
+      description "Minimum LSP regeneration interval.";
+      leaf level-1 {
+        type uint16 {
+          range "1..120";
+        }
+        units "seconds";
+        configd:help "Minimum time allowed before level-1 LSP retransmissions.";
+        description "Minimum time allowed before level-1 LSP retransmissions.";
+      }
+      leaf level-2 {
+        type uint16 {
+          range "1..120";
+        }
+        units "seconds";
+        configd:help "Minimum time allowed before level-2 LSP retransmissions.";
+        description "Minimum time allowed before level-2 LSP retransmissions.";
+      }
+    }
+    container lsp-refresh-interval {
+      description "LSP refresh interval";
+      configd:help "LSP refresh interval";
+      leaf level-1 {
+        type uint16;
+        units "seconds";
+        configd:help "LSP refresh interval for level-1.";
+        description "LSP refresh interval for level-1.";
+        must ". > ../../lsp-gen-interval/level-1" {
+          error-message "lsp refresh interval must be greater than lsp general interval";
+        }
+      }
+      leaf level-2 {
+        type uint16;
+        units "seconds";
+        configd:help "LSP refresh interval for level-2.";
+        description "LSP refresh interval for level-2.";
+        must ". > ../../lsp-gen-interval/level-2" {
+          error-message "lsp refresh interval must be greater than lsp general interval";
+        }
+      }
+    }
+
+    container max-lsp-lifetime {
+      configd:help "Maximum LSP lifetime.";
+      description "Maximum LSP lifetime.";
+      leaf level-1 {
+        type uint16 {
+          range "350..65535";
+        }
+        units "seconds";
+        configd:help "Maximum LSP lifetime for level-1.";
+        description "Maximum LSP lifetime for level-1.";
+      }
+      leaf level-2 {
+        type uint16 {
+          range "350..65535";
+        }
+        units "seconds";
+        configd:help "Maximum LSP lifetime for level-2.";
+        description "Maximum LSP lifetime for level-2.";
+      }
+    }
+
+    leaf set-overload-bit {
+      type empty;
+      configd:help "If true, avoid any transit traffic.";
+      description "If true, avoid any transit traffic.";
+    }
+
+    container spf-delay-ietf {
+      presence "Present if IETF SPF back-off delay is enabled.";
+      configd:help "IETF SPF delay algorithm.";
+      description "IETF SPF delay algorithm.";
+      list init-delay {
```

Comment:
Please could you share the reasoning for using the nested set of lists? IMO it would be simpler and cleaner to have all these delay values as leaves at the top level of the container. Doing this would also prevent the user from configuring multiple of the same delay value (which can be done with lists, using `max-elements`, but I think using leaves at the top level is a more straightforward and readable approach).

Reply:
Got it. Thanks @deastoe . 
This is Resolved.